clang/include/clang/StaticAnalyzer/Core/AnalyzerOptions.def

```diff
@@ -189,20 +189,23 @@ ANALYZER_OPTION(
     "crosscheck-with-z3-eqclass-timeout-threshold",
     "Set a timeout for bug report equivalence classes in milliseconds. "
     "If we exhaust this threshold, we will drop the bug report eqclass "
-    "instead of doing more Z3 queries. Set 0 for no timeout.", 700)
+    "instead of doing more Z3 queries. On fast machines, 700 is a sane value. "
+    "Set 0 for no timeout.", 0)
 
 ANALYZER_OPTION(
     unsigned, Z3CrosscheckTimeoutThreshold,
     "crosscheck-with-z3-timeout-threshold",
-    "Set a timeout for individual Z3 queries in milliseconds. "
-    "Set 0 for no timeout.", 300)
+    "Set a timeout for individual Z3 queries in milliseconds. On fast "
+    "machines, 400 is a sane value. "
+    "Set 0 for no timeout.", 15'000)
```

Comment:
Why did you mention 400 instead of 300  in `On fast machines, 400 is a sane value.`
I also believe that to promote insensitivity, we should avoid using the word "sane".
I think I've read that on [Google dev doc style](https://developers.google.com/style/inclusive-documentation), mentioned on [LLVM discourse](https://discourse.llvm.org/t/inclusive-language-in-llvm-can-we-rename-master-branch/55675/68.
```suggestion
    "Set a timeout for individual Z3 queries in milliseconds. "
    "On fast machines, 300 worked well in some cases. "
    "The lower it is, the higher the chances of having flaky issues. "
    "Having no timeout may hang the analyzer indefinitely. "
    "Set 0 for no timeout.", 15'000)
```